src/webui/static/app/epg.js

```diff
@@ -1050,7 +1060,7 @@ tvheadend.epg = function() {
                    state.filters = {};
                    /* Otherwise this non-resizable column will not expand to newly set width */
                    if (state.columns)
-                       state.columns[0].width = actions.width;
+                       state.columns[0].width = eventdetails.width;
```

Comment:
I wasn't 100% certain on this one - I've renamed the object, so all works, but I wasn't certain whether we should do the same with eventactions.width. I certainly couldn't get it to do anything, but my Javascript isn't up to working out why/how (or even whether...).